execution.bs

```diff
@@ -4819,6 +4819,27 @@ enum class forward_progress_guarantee {
 1. For the purposes of this subclause, a sender is an object that satisfies the
     `sender` concept ([async.ops]).
 
+2. Subclauses [exec.factories] and [exec.adapt] define customizable algorithms
+    that return senders. Each algorithm has a default implementation specified
+    herein. Let `sndr` be the result of an invocation of such an algorithm or an
+    object equal to such ([concepts.equality]), and let `Sndr` be
+    `decltype((sndr))`. Let `Env` be the type of an environment `env` such that
+    `sender_in<Sndr, Env>` is `true`, and let `rcvr` be a receiver whose
+    associated environment is `Env`. For the default implementation of the
+    algorithm that produced `sndr`, connecting `sndr` to `rcvr` and starting the
+    resulting operation state ([async.ops]) necessarily results in the potential
+    evaluation ([intro.execution]) of a set of completion operations whose first
+    argument is a subexpression equal to `rcvr`. Let `Sigs` be a pack of
+    completion signatures corresponding to this set of completion operations.
+    Then the type of the expression `get_completion_signatures(sndr, env)` is a
+    specialization of the class template `completion_signatures`,
+    ([exec.utils.cmplsigs]) the set of whose template arguments is `Sigs`. If a
+    user-provided implementation of the algorithm that produced `sndr` is
+    selected instead of the default, the set of completion signatures denoted by
+    the return type of `get_completion_signatures(sndr, env)` shall be a
+    superset of `Sigs`, with any additional completion signatures corresponding
+    to error or stopped completion operations.
```

Comment:
```suggestion
    selected instead of the default, any completion signature from a set of completion signatures denoted by
    the return type of `get_completion_signatures(sndr, env)` that is not part of `Sigs`, shall correspond
    to error or stopped completion operation.
```

Reply:
```suggestion
    selected instead of the default, any completion signature in the set of types denoted by `completion_signatures_of_t<Sndr, Env>` that is not part of `Sigs`, shall correspond
    to error or stopped completion operations.
```